.gitignore

```diff
@@ -0,0 +1,24 @@
+config.json
+
+# Logs
+logs
+*.log
+npm-debug.log*
+
+# Runtime data
+pids
+*.pid
+*.seed
```

Comment:
I'm keen to keep gitignore as clean as possible and only add to it when we need. At the moment I feel like the only items we need are

```
.DS_Store
node_modules
npm-debug.log
config.json
```

I haven't run in to issues with `.npm` or `.node_repl_history` before though.